python/lsst/cp/pipe/utils.py

```diff
@@ -171,8 +171,8 @@ def makeMockFlats(expTime, gain=1.0, readNoiseElectrons=5, fluxElectrons=1000,
         flatExp1.image.array[:] = flatData1/gain   # ADU
         flatExp2.image.array[:] = flatData2/gain   # ADU
 
-    visitInfoExp1 = lsst.afw.image.VisitInfo(exposureId=expId1, exposureTime=expTime)
-    visitInfoExp2 = lsst.afw.image.VisitInfo(exposureId=expId2, exposureTime=expTime)
+    visitInfoExp1 = lsst.afw.image.VisitInfo(exposureTime=expTime)
+    visitInfoExp2 = lsst.afw.image.VisitInfo(exposureTime=expTime)
```

Comment:
Should this one use `id`? Seems a bit odd that it used to need an ID, but now it doesn't.

Reply:
The ids are set in the ExposureInfo just below, so the ids are preserved. It's not clear from context whether these were supposed to be detector+visit ids, or just visit ids, but looking at the use of this (in tests), it probably doesn't matter either way.

Reply:
I also vaguely recall the exposureId field being the only required field early on, so this might be vestigial from that.